config/secure-headers.php

```diff
@@ -380,38 +560,17 @@
 				'blob:',
 			],
 		],
-		'default-src' => [
-			'self' => true,
-			'allow' => [
-				'blob:',
-			],
-		],
-		'base-uri' => [],
-		'connect-src' => [
-			'allow' => [
-				'http://lycheeorg.github.io/update.json',
-				'blob:',
-			],
-			'self' => true,
-		],
-		'form-action' => [
+
+		'style-src' => [
 			'self' => true,
+			'unsafe-inline' => true, // We sadly need this one
 		],
-		'frame-ancestors' => [],
-		'frame-src' => [],
-		'manifest-src' => [],
-		'media-src' => [
+
+		'default-src' => [
 			'self' => true,
 			'allow' => [
 				'blob:',
 			],
```

Comment:
```suggestion
			'allow' => [],
```

Reply:
Removed in [3f5583f](https://github.com/LycheeOrg/Lychee/pull/1528/commits/3f5583f3f4300551dbb90229ed1d7e908202272d)